README.md

```diff
@@ -278,36 +222,87 @@ Check out `tune --help` for all possible CLI commands and options. For more info
+Then, a finetune can be kicked off:
 
-### Simplicity and Extensibility
+```bash
+tune run --nproc_per_node 8 lora_finetune_distributed --config llama3_1/70B_lora.yaml
+```
 
-torchtune is designed to be easy to understand, use and extend.
+You can find a full list of all our Llama3 configs [here](recipes/configs/llama3) and Llama3.1 configs [here.](recipes/configs/llama3_1)
 
-- Composition over implementation inheritance - layers of inheritance for code re-use makes the code hard to read and extend
-- No training frameworks - explicitly outlining the training logic makes it easy to extend for custom use cases
-- Code duplication is preferred over unnecessary abstractions
-- Modular building blocks over monolithic components
+---
 
-### Correctness
+## Community
 
-torchtune provides well-tested components with a high-bar on correctness. The library will never be the first to provide a feature, but available features will be thoroughly tested. We provide
+torchtune focuses on integrating with popular tools and libraries from the ecosystem. These are just a few examples, with more under development:
 
-- Extensive unit-tests to ensure component-level numerical parity with reference implementations
-- Checkpoint-tests to ensure model-level numerical parity with reference implementations
-- Integration tests to ensure recipe-level performance parity with reference implementations on standard benchmarks
+- [Hugging Face Hub](https://huggingface.co/docs/hub/en/index) for [accessing model weights](torchtune/_cli/download.py)
+- [EleutherAI's LM Eval Harness](https://github.com/EleutherAI/lm-evaluation-harness) for [evaluating](recipes/eleuther_eval.py) trained models
+- [Hugging Face Datasets](https://huggingface.co/docs/datasets/en/index) for [access](torchtune/datasets/_instruct.py) to training and evaluation datasets
+- [PyTorch FSDP2](https://github.com/pytorch/torchtitan/blob/main/docs/fsdp.md) for distributed training
+- [torchao](https://github.com/pytorch-labs/ao) for lower precision dtypes and [post-training quantization](recipes/quantize.py) techniques
+- [Weights & Biases](https://wandb.ai/site) for [logging](https://pytorch.org/torchtune/main/deep_dives/wandb_logging.html) metrics and checkpoints, and tracking training progress
+- [Comet](https://www.comet.com/site/) as another option for [logging](https://pytorch.org/torchtune/main/deep_dives/comet_logging.html)
+- [ExecuTorch](https://pytorch.org/executorch-overview) for [on-device inference](https://github.com/pytorch/executorch/tree/main/examples/models/llama2#optional-finetuning) using fine-tuned models
+- [bitsandbytes](https://huggingface.co/docs/bitsandbytes/main/en/index) for low memory optimizers for our [single-device recipes](recipes/configs/llama2/7B_full_low_memory.yaml)
+- [PEFT](https://github.com/huggingface/peft) for continued fine-tuning or inference with torchtune models in the Hugging Face ecosystem
 
 &nbsp;
 
-## Community Contributions
+### Community Contributions
```

Comment:
are we keeping this? I remember we had a discussion about it a while ago

Reply:
Yeah I planned to keep it for now